### ISSUE TITLE
Meta field with data being a collection

### ISSUE THREAD

**Issue by fabricehong** [jasminb/jsonapi-converter#15]
Hello,

it seems impossible to me to parse meta information when the json data is a collection.

How to parse meta info for a JSON such as :

```
{
  "data": [
    {
      "id": "11",
      "type": "address",
      "attributes": {
        "city": "Lausanne",
        ...
      }
    },
    {
      "id": "12",
      "type": "address",
      "attributes": {
        "city": "Bern",
        
	...
      }
    }
  ],
  "jsonapi": {
    "version": "1.0"
  },
  "meta": {
    "default_billing_address_id": "11",
    "default_shipping_address_id": "12"
  }
}
```

Do you see a solution ?

cheers

**Comment by jasminb** [comment]
Hey,

Simple solution would be to have redundant meta on each collection element.

One could define meta attribute on collection type and than change collection conversion logic to set meta object to each of the elements.

Jasmin

**Comment by fabricehong** [comment]
That's a possible solution. But the server sending an empty array of object doesn't mean that it won't send valuable meta information.

Actually your solution fits my usecase.. And since it seems there is no easy solution to this, I could implement it as a temporary solution, and make a PR

**Comment by jasminb** [comment]
Hey,

Will need to give this some more thought.

Regards,
Jasmin

**Comment by jasminb** [comment]
Implemented and merged on develop with #41
